assets/styles/style.css

```diff
@@ -401,23 +401,6 @@ PLUGIN STYLES
   background-position: 3px 4px;
 }
 
-@media (max-width:767px) {
-  .menu-item {
-    width: 30px;
-    margin: 0 5px;
-    overflow: hidden;
-    color: transparent;
-  }
-}
-
-.header-grid {
```

Comment:
The `header-grid` class is the main container class for the header, so I think it makes sense to have it at the top of the header section. Please move it back up here.